content/downloads/_index.md

```diff
@@ -9,14 +9,16 @@ menu:
 # Downloads
 
 ### Presentations from FHIR DevDays, November 2020
-* <a href="201118_Mark_Kramer_Introduction_To_FHIR_Shorthand.pptx">Introduction to FHIR Shorthand</a>
-* <a href="201118_Chris_Moesel_Lets_Build_with_FHIR_Shorthand.pptx">Let's Build with FHIR Shorthand</a>
+
+* <a href="201118_Mark_Kramer_Introduction_To_FHIR_Shorthand.pdf">Introduction to FHIR Shorthand</a>
+* <a href="201118_Chris_Moesel_Lets_Build_with_FHIR_Shorthand.pdf">Let's Build with FHIR Shorthand</a>
 
 ### Tutorials from the HL7 FHIR Connectathon, September 2020
-* <a href="sept-2020-fsh-tutorial-part-1.pptx">Introduction to FHIR Shorthand</a>
-* <a href="sept-2020-fsh-tutorial-part-2.pptx">Advanced FHIR Shorthand</a>
-* <a href="sept-2020-fsh-tutorial-part-3.pptx">Wicked FHIR Shorthand (Version 2 Plans)</a>
-* <a href="sept-2020-fshing-equipment.pptx">FSHing Equipment (Tools)</a>
+
+* <a href="sept-2020-fsh-tutorial-part-1.pdf">Introduction to FHIR Shorthand</a>
```

Comment:
Broken link.  The PDF you uploaded is named `FHIR Shorthand Tutorial - Part 1 - September 2020.pdf `.

Reply:
Stupid stupid stupid! Fixed.